src/Menu/Menu.scss

```diff
@@ -0,0 +1,51 @@
+$calorieTagColor: #49a0b5;
+$typeTagColor: #2e7cf6;
+$sideTagColor: #f7c244;
```

Comment:
the way these colours are named, they definitely would not be reusable on other elements. i think these colours are from bootstrap anyway? is there any way you could apply these colours using bootstrap instead of defining them yourselves?

Reply:
also variables for colours should go in the file _variables.scss which already exists i think

Reply:
We didn't use the bootstrap colors. However following your various comments about the way colours are defined, we chose to declare a set of colours as variables in _variables.scss and to use them throughout the whole app